### PR TITLE
Change print statements to log.info

### PR DESCRIPTION
**Pull request by harish-24**
Patch replaces print statements to log.info so that we can avoid both python2 and python3 compatibility

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

File: io/net/tcpdump.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def test(self):
             if "packets dropped by interface" in line:
                 if int(line[0]) >= (int(self.drop) * int(self.count) / 100):
                     self.fail("%s, more than %s percent" % (line, self.drop))
-                print line
+                self.log.info(line)
```
Comment (narasimhan-v) [inline io/net/tcpdump.py]: @harish-24 though this should not be part of this commit, can you change it like:
```
            if "packets dropped by interface" in line:
                self.log.info(line)
                if int(line[0]) >= (int(self.drop) * int(self.count) / 100):
                    self.fail("%s, more than %s percent" % (line, self.drop))
```

Comment (harish-24) [reply io/net/tcpdump.py]: @narasimhan-v Done.
